src/app/my-payment-methods/my-payment-methods.component.html

```diff
@@ -133,6 +133,17 @@ <h3>
           size="medium"
           url="/my-account/payment-methods/change-regular-giving"
         />
+
+        <!-- not necessarily needed for regular giving MVP but would be good to have a busy state option built
```

Comment:
In other places we're using conditionals and a (non-Components) spinner for this. So maybe it would work better as an Angular component wrapping that logic?

Reply:
Yes that might work, we'd have to get out of the habit of directly using our component button in angular then, at least for anything that might require user to wait.

I guess even nicer than a wait state that's manually assigned might be if we could pass the component an async onclick function and it would automatically change its appearance before and after running that function.